tests/datasets/test_entity_factory.py

```diff
@@ -29,6 +29,7 @@
     EntityKey.PROFILES,
     EntityKey.FUNCTIONS,
     EntityKey.REPLAYS,
+    EntityKey.REPLAYS_AGGREGATED,
```

Comment:
will this test against the clickhouse table?

Reply:
I have no idea what this test does.  But now that the YAML is defined the test requires it.

Reply:
kk just wondering because I def did not review every column of the schema lol